src/ModelerApp.vue

```diff
@@ -8,13 +8,11 @@
       </div>
     </div>
     <div class="modeler-container">
-      <modeler ref="modeler" />
+      <modeler ref="modeler" @validate="validationErrors = $event" />
     </div>
     <statusbar>
-      {{statusText}}
-      <font-awesome-icon :style="{color: statusColor}" :icon="statusIcon" />
+      <validationStatus :validation-errors="validationErrors"/>
```

Comment:
The other components in this file are named using kebab-case (`modeler`, `b-modal`, `file-upload`); I think this should follow the same convention: `validation-status`.